src/Translator/BCLabelTranslatorStrategy.php

```diff
@@ -18,7 +18,7 @@
  */
 final class BCLabelTranslatorStrategy implements LabelTranslatorStrategyInterface
 {
-    public function getLabel($label, $context = '', $type = '')
+    public function getLabel(string $label, string $context = '', string $type = ''): string
```

Comment:
This is a final class, which means the return type hint could be contributed to 3.x.

Please open a PR with the backwards-compatible part of your fixes on 3.x :pray:

Reply:
The classes are not final on 3.x @greg0ire